zerver/migrations/0458_remove_invalid_characters_from_system_group_name.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 4.2.2 on 2023-07-03 06:39
+
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def remove_invalid_characters_from_system_group_name(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    # Remove "@" character from name of role-based system groups.
+    UserGroup = apps.get_model("zerver", "UserGroup")
+
+    groups_to_update = []
+    for group in UserGroup.objects.filter(is_system_group=True):
+        old_group_name = group.name
+        group.name = old_group_name[1:]
```

Comment:
Do we want to have a check here? In the rare case when this migration gets run twice, valid user group names get truncated as well.

Reply:
I added the check to be on the safer side.